Assets/Scripts/ChallengeManagerScript.cs

```diff
@@ -89,11 +89,13 @@ void Start()
 
         // Display instruction panel
         DisplayChallengeInstructions();
+
         MusicManagerScript.Instance.TurnOffEverything();
         MusicManagerScript.Instance.whichSource += 1;
         MusicManagerScript.Instance.whichSource = MusicManagerScript.Instance.whichSource % 2;
         MusicManagerScript.Instance.SwitchToSource();
-        MusicManagerScript.Instance.StartMusic();
+		MusicManagerScript.Instance.StartRoot();
```

Comment:
Curious how come this shows as changed

Reply:
Oops didn't mean to commit this. Was trying to get music working on that screen and it didn't work.